src/components/userProfile/UserProfile.js

```diff
@@ -276,31 +304,42 @@ class UserProfile extends Component {
       );
     }
 
+    let { scale } = this.state;
+    let imageStyle = { transform: [{ scale }] };
+
     return (
       <View style={{
         justifyContent: 'space-between',
         flexDirection: 'column',
         flex: 1,
       }}>
-        <ScrollView>
+        <ScrollView
+            scrollEventThrottle={16}
+            onScroll={this.handleScroll.bind(this)}>
```

Comment:
The closing bracket must be aligned with the line containing the opening tag (expected column 9 on the next line)